please.sh

```diff
@@ -116,6 +116,7 @@ sdk_path () { # <bitness>
 
 sdk64="$(sdk_path 64)"
 sdk32="$(sdk_path 32)"
+sdkarm64="$(sdk_path arm64)"
```

Comment:
I fear that we'll have to do some major cleanup here.

The original design was for the `please.sh` script to be called in a dedicated VM that was the home of both i686 and x86_64 SDK, and both should be handled in synchrony. For example: whenever I built `msys2-runtime`, it was to be built in both SDKs, and installed in both SDKs. That's why there is an `sdk_path` to begin with.

However, we're safely outside this original scenario. Long gone is the time that I even ran VMs on my own machine.

And only on a Windows/ARM64 machine could we even _use_ side-by-side i686/x86_64/aarch64 SDKs. But I don't have such a machine (except a relatively slow Azure VM that I only power up when I need to work on Windows/ARM64 stuff).

The common case, therefore, is that the "sdk path" is `/`.

So I think that soon we will need to do some major spring cleaning and rip out this idea that more than one SDK is available, and that one should be "callable" from another one. Read: we will want to get rid of `sdk_path`, `sdk64`, `sdk32` and `sdkarm64` as well as all of their users.